### PR TITLE
SB-1083: Memory OrientDB Storage for unit tests

### PR DESCRIPTION
**Pull request by sbespalov**

### CODE REVIEW

File: strongbox-data-service/src/main/resources/orientdb_MEMORY.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+strongbox.orientdb.protocol = memory
+strongbox.orientdb.database = strongbox
+strongbox.orientdb.username = admin
+strongbox.orientdb.password = password
```
Comment (carlspring) [inline strongbox-data-service/src/main/resources/orientdb_MEMORY.properties]: Please, make sure there are empty lines at the end of such resource files. (Here and below).  :)

Comment (sbespalov) [reply strongbox-data-service/src/main/resources/orientdb_MEMORY.properties]: done

**Comment by sbespalov** [comment]
@carlspring I have checked it with distribution and it works fine

File: strongbox-data-service/src/main/resources/orientdb_MEMORY.properties (hunk 1 above)
Comment (carlspring) [reply strongbox-data-service/src/main/resources/orientdb_MEMORY.properties]: Thanks! :)

**Comment by carlspring** [comment]
Thanks, Sergey!

**Pull request merged**